lib/ruby_lsp/document.rb

```diff
@@ -81,7 +90,6 @@ def update_parsable_source(edits)
       # If the new edits caused a syntax error, make all edits blank spaces and line breaks to adjust the line and
       # column numbers. This is attempt to make the document parsable while partial edits are being applied
       edits.each do |edit|
-        @syntax_error_edits << edit
         next if edit[:text].empty? # skip deletions, since they may have caused the syntax error
 
         apply_edit(@parsable_source, edit[:range], edit[:text].gsub(/[^\r\n]/, " "))
```

Comment:
What is the difference between `@source` and `@parsable_source`/What is `@parsable_source` being used for here?